biopandas/pdb/pandas_pdb.py

```diff
@@ -5,23 +5,27 @@
 # License: BSD 3 clause
 # Project Website: http://rasbt.github.io/biopandas/
 # Code Repository: https://github.com/rasbt/biopandas
+from __future__ import annotations
```

Comment:
I don't think this is necessary. I think type annotations were introduced in Python 3.6. If someone has an older version of Python, they won't be able to run it anyways because of the f-strings?

Reply:
No worries, I can take care of removing it.